common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -29,6 +29,11 @@
 public final class CommonUtils {
   private static final Logger LOG = LoggerFactory.getLogger("");
 
+  /**
+   * Get the current time in milliseconds.
+   *
+   * @return current time in milliseconds
+   */
```

Comment:
Would this function count as a getter though. In my mind, getter / setter is a getFoo / setFoo member method that gets / sets the value of the member field mFoo.